release_notes.md

```diff
@@ -2,8 +2,8 @@
 
 #### Host Version
 
-- Host Version: 4.1041.200
-- In-Proc Host Version: 4.41.100 (4.841.100, 4.641.100)
+- Host Version: 4.1042.100
```

Comment:
You only had to change the host version to `4.1042.100, please add `In-Proc Host Version` back and leave it on the old version